Sloth.Sql/dbo/Tables/TransactionStatusEvents.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE [dbo].[TransactionStatusEvents]
+(
+    [Id] NVARCHAR(450) NOT NULL PRIMARY KEY, 
+    [TransactionId] NVARCHAR(450) NOT NULL,
+    [Status] NVARCHAR(50) NOT NULL, 
+    [EventDate] DATETIME2 NOT NULL, 
+    [EventDetails] NVARCHAR(450) NOT NULL, 
+    CONSTRAINT [FK_TransactionStatusEvents_Transactions] FOREIGN KEY ([TransactionId]) REFERENCES [Transactions]([Id])
+)
+
+GO
+
+CREATE NONCLUSTERED INDEX [IX_TransactionStatusEvents_TransactionId] ON [dbo].[TransactionStatusEvents] ([TransactionId] ASC)
+
+GO
+
+CREATE NONCLUSTERED INDEX [IX_TransactionStatusEvents_Status] ON [dbo].[TransactionStatusEvents] ([Status] ASC)
+
+GO
+
+CREATE NONCLUSTERED INDEX [IX_TransactionStatusEvents_EventDate] ON [dbo].[TransactionStatusEvents] ([EventDate] ASC)
+
+GO
+
+CREATE NONCLUSTERED INDEX [IX_TransactionStatusEvents_EventDetails] ON [dbo].[TransactionStatusEvents] ([EventDetails] ASC)
```

Comment:
Do we really need indexes on every field?  obviously the transactionID is good as a PK but I'm not sure what scenario it's useful to be able to search on the details or even date

Reply:
@srkirkland the index on details made more sense before I changed what it was storing. I'll remove it. Kinda want to keep the one on EventDate though.